### ISSUE TITLE
Register FastMonitoringService as concrete type to Service system

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#44942]
Everywhere that request the MicroStateService from `edm::Service` eventually does a C++ cast to `FastMonitoringService*`. It would be much better to just register the service as `FastMonitoringService` instead of as the base class.

**Comment by Dr15Jones** [comment]
assign EventFilter/Utilities

**Comment by Dr15Jones** [comment]
The `MicroStateService` class is empty (i.e. no member function or member data). That entire class should just go away. 

Or much, much better `MicroStateService` should gain abstract virtual functions for all the methods modules use from `FastMonitoringService` and then all module should just talk through those virtual methods. Then `FastMonitoringService` should move to the _plugins_ directory. This would allow FWCore/ActivityRegistry dependency to be dropped from EventFilters/Utilities which would decrease our overall physical dependencies in or libraries.

**Comment by smorovic** [comment]
Agreed, MicroStateService is legacy class that should be dropped, so we will prepare these changes.
To drop FWCore/ActivityRegistry dependency, I assume that we also need to move EvFDaqDirector to plugins.
